src/FeatureManager.php

```diff
@@ -178,17 +179,19 @@ public function createDatabaseDriver(array $config, string $name)
      * Serialize the given scope for storage.
      *
      * @param  mixed  $scope
+     * @param  string  $driver
      * @return string|null
      */
-    public function serializeScope($scope)
+    public function serializeScope($scope, $driver = '')
     {
         return match (true) {
+            $scope instanceof FeatureScopeSerializeable => $scope->featureScopeSerialize($driver),
```

Comment:
I personally feel that passing the driver here is redundant if `FeatureScopeable` will remain available.

My thinking is that FeatureScopeable can be used in combination with FeatureScopeSerializable to replace this.

Reply:
I can get behind that.